library/core/src/main/java/com/google/android/exoplayer2/video/VideoFrameReleaseTimeHelper.java

```diff
@@ -68,19 +74,12 @@ public VideoFrameReleaseTimeHelper() {
    */
   public VideoFrameReleaseTimeHelper(Context context) {
     this(getDefaultDisplayRefreshRate(context));
+    this.context = context;
```

Comment:
Use `context.getApplicationContext()` to ensure the class can't leak any shorter lived contexts that might be passed to it (it's always fine to hold onto the application context, since it's expected to be around for as long as the process remains alive).